doc/examples/nearby_places/NearbyPlacesQueryType.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: bdunogier
+ * Date: 29/09/2018
+ * Time: 16:46
+ */
+
+namespace AppBundle\QueryType;
```

Comment:
```suggestion
namespace App\QueryType;
```

The examples needs be updated to SF4 as well.